code/game/objects/structures/grille.dm

```diff
@@ -141,6 +141,8 @@
 		return
 	var/mob/M = AM
 	shock(M, 70)
+	if(prob(50))
+		take_damage(1, BRUTE, "melee", FALSE)
```

Comment:
what's brute melee damage about this? also use named arguments

Reply:
I'm using someone else's code, you'd have to ask them why they did it that way. Can you suggest a change that'd put it in line with how you're wanting it? I'll implement it however you think is best.